cpp/search/searchparams.h

```diff
@@ -49,7 +49,7 @@ struct SearchParams {
   double rootFpuReductionMax; //Same as fpuReductionMax, but at root
   double rootFpuLossProp; //Same as fpuLossProp, but at root
   int rootNumSymmetriesToSample; //For the root node, sample this many random symmetries (WITHOUT replacement) and average the results together.
-
+  bool rootSearchSymmetriesReduce; //For the root node, if the board is symmetric, only search one loc among its symmetric equivalence.
```

Comment:
Very minor: Correct grammar would be "For the root node, search only one copy of each symmetrically equivalent move".

Reply:
Done